v2x_ros_driver/include/v2x_ros_driver/v2x_radio_client.h

```diff
@@ -123,7 +123,27 @@ class V2XRadioClient
     /**
      * @brief set wave config path
     */
-   void set_wave_file_path(const std::string &path);
+    void set_wave_file_path(const std::string &path);
+
+    /**
+     * @brief Checks actual message vector size against size specified in MessageFrame
+     */
+    bool isValidMsgSize(const std::vector<uint8_t> msg_vec, size_t start_index, size_t end_index, const std::vector<uint8_t> entry);
```

Comment:
While this file was made by an external company and does not contain additional comments for functions, would you mind adding comments for the function arguments as well for these new functions? Similar to [this example in basic_autonomy](https://github.com/usdot-fhwa-stol/carma-platform/blob/7888a6e80c16f55383866e9102085433c8d490ed/basic_autonomy/include/basic_autonomy/basic_autonomy.hpp#L122)

Reply:
Added param descriptions